### PR TITLE
Updates Desktop 1.1.2-beta5 change log

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
This is for getting the final beta out before we ship 1.2.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.1.2-beta5": [
+      "[Added] Syntax highlighting for java server pages (jsp) - #4470. Thanks @damaneice!",
+      "[Fixed] Ahead/behind info is not remembered when selecting branch to compare  - #4517",
```
Comment (nerdneha) [inline changelog.json]: i'd be open to making this an "Improved" since it wasn't necessarily wrong but we're making an optimization. thoughts?

Comment (iAmWillShepherd) [reply changelog.json]: Yea, I can get behind marking that as an improvement.

**Comment by iAmWillShepherd** [comment]
💯

**Comment by iAmWillShepherd** [comment]
💯

**Comment by shiftkey** [comment]
@nerdneha I'm gonna #jolo this in so that we're not holding up the release

**Pull request merged**